src/components/ui/Badge/DiscountBadge.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import { useDiscountPercent } from 'src/sdk/product/useDiscountPercent'
+
+import Badge from './Badge'
+
+import './badge.scss'
+
+type Props = {
+  listPrice: number
+  spotPrice: number
+  small?: boolean
+  thresholdLow?: number
```

Comment:
Let's add some comment here to help people understand it better. What do you think?

How about something like this:
- thresholdLow
// set limit percentage value to consider a low discount

- thresholdHigh
// set limit percentage value to consider a high discount